src/language/error.test.ts

```diff
@@ -452,4 +452,35 @@ df'
       });
     });
   });
+
+  describe('map parser', () => {
+    it('should return correct error with wrong return syntax in operations', () => {
+      const input = `profile = "pro/file@1.0"
+      provider = "provider"
+
+      operation Foo {
+        http GET "url" {
+          response 200 {
+            return map result {
+              results = 1
+            }
+          }
+        }
+      }
+      `;
+
+      const source = new Source(input);
+
+      expect(() => parseMap(source)).toThrowSyntaxError(
+        "Error in script syntax: ',' expected.",
```

Comment:
It's a ts compiler suggestion - it sees `map result { /* ... */ }` and tries to parse it as an expression. This itself is not valid syntax, so it suggets `map, result { /* ... */ }` because that makes it less invalid - in js `1, 2, 3` is an expression that evaluates all expressions separated by commas (in this case `1` and `2` and `3`) and returns the value of the last one (`3`).

The actuall error is in the comlink syntax being violated but right now I don't have a clear idea how to work with that.